### ISSUE TITLE
Feature request: How to customise the commit message format ?

### ISSUE THREAD

**Issue by mohamedfasil** [ngryman/cz-emoji#31]
How to customise the commit message format ?
Is there any option to provide it in the config ?
for eg:-
```
"config": {
    "commitizen": {
      "path": "cz-emoji",
      "cz-emoji": {
        "format": "[name] [scope]: [emoji] [subject]"
      }
    }
  }
```
gives `feat (scope): ✨ new feature`

**Comment by mohamedfasil** [comment]
have opened a PR #32   @ngryman

**Comment by ngryman** [comment]
Fixed in #66.
